### ISSUE TITLE
Allow currency selection on "Send" screen

### ISSUE THREAD

**Issue by danfinlay** [MetaMask/metamask-extension#3415]
I know we've talked about this again, but I think it's worth revisiting.

Also suggested again by a user in #3413, proposed with mockups in [this google doc](https://docs.google.com/document/d/1MUEz2Ioif31zzELBTfmNyCTG7_W--QWoIkr7mU0cQ7k/edit).

**Comment by bdresser** [comment]
probably needs to come after #3824

**Comment by snreynolds** [comment]
If we could allow token selection on the send screen, it could give us room to add the token balances on that screen as well as mentioned in #4772

**Comment by danjm** [comment]
@danfinlay can this be closed in favour of #4772 / #4903? Or is this a suggestion that we should allow users to select on the send screen the fiat currency that their ETH / tokens are converted to?

**Comment by danfinlay** [comment]
Nope, it's the asset being sent. Closing in favor of #4772/#4903.

**Comment by sukuroy** [comment]
Do we have the token selection functionality now? Currently, only ETH shows up during the transfer option. This would mean that if I have other tokens and 0 ETH then I get a prompt saying 'insufficient funds'

**Comment by bdresser** [comment]
@sukuroy this functionality isn't in production yet. The error message you mention is fixed in https://github.com/MetaMask/metamask-extension/pull/4694, and #4772 tracks allowing the user to swap the token they're sending from the currency screen itself.
